services/alertmanager/alertmanager.go

```diff
@@ -50,6 +55,13 @@ type Service struct {
 
 // New creates new service.
 func New(db *reform.DB, alertsRegistry *Registry) *Service {
+	l := logrus.WithField("component", "alertmanager")
+
+	if d, err := time.ParseDuration(os.Getenv(envInterval)); err == nil && d > 0 {
+		l.Warnf("Interval changed to %s.", d)
+		resendInterval = d
```

Comment:
`New` "constructor" should not modify non-local state. Make resendInterval a field.